index.html

```diff
@@ -199,7 +199,21 @@ <h2 class="subt">Skills</h2>
           </div>
         </div>
       </section>
-    </main>
-    <footer></footer>
+      <footer class="contact-form">
+        <section class="form-section">
+          <div class="contact-me">
+            <h2 >Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form action="send" class="form">
+            <input type="text" class="text" value="Yeremias">
```

Comment:
Please add the HTML validations to your inputs. For example, the name should have a maxlength of 30. Kindly apply this change to the other fields but with their specific requirements 🙌

Reply:
Modifications done!